src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -1607,9 +1615,21 @@ public void updateCounts( final MoleculeCounts moleculeCounts,
         return new Interval(readStart, readEnd);
     }
 
-    private static ReadReport combineReports( final ReadReport report1, final ReadReport report2 ) {
-        if ( report1.getRefCoverage().isEmpty() ) return report2;
-        if ( report2.getRefCoverage().isEmpty() ) return report1;
-        return new PairedReadReport(report1, report2);
+    @VisibleForTesting static void combineReports( final ReadReport report1, final ReadReport report2 ) {
```

Comment:
First, I understand that this absorbs this the functionality of the old `PairedReadReport.updateCounts(...)`.

I find this new architecture a little inconsistent: before this PR, there's the class `ReadReport`, and `PairedReadReport`, which now are merged into a single `final` class `ReadReport`. Now in this new `ReadReport`, there's a ctor that takes in two `ReadReport`'s. A reader would be confused by these two: why are there two functions essentially having the same signature (sure, return types are different)? 

So I'd recommend looking for a solution to absorb this into the class `ReadReport` too.

Reply:
> Maybe combineReports should be renamed to updateCountsForPair? Would that be better?

Yep, I think this name reflects the intention better.